cloudslang-cli/src/test/java/io/cloudslang/lang/cli/utils/CompilerHelperTest.java

```diff
@@ -101,6 +104,48 @@ public void testDependenciesFileParentFolder() throws Exception {
                         SlangSource.fromFile(flowPath)
                 )
         );
+        Mockito.verify(slang).compileCleanUp();
+    }
+
+    @Test
+    public void testCompileSourceCleanup() throws Exception {
+        URI flowPath = getClass().getResource("/executables/dir3/flow.sl").toURI();
+        URI opPath = getClass().getResource("/executables/dir3/dir3_1/test_op.sl").toURI();
+        compilerHelper.compileSource(flowPath.getPath(), null);
```

Comment:
i think we should have inOrder verification between `compilerHelper.compileSource` and `slang.compileCleanup`